app/views/menu/help-menu.tmpl.html

```diff
@@ -0,0 +1,21 @@
+<span role="help-button">
```

Comment:
Just a note, disagree to `Do Not Capitalize Every Word` as it is common practice and you see that very often. :)